INSTALL.md

```diff
@@ -0,0 +1,43 @@
+# Installation instructions
+
+Given that there have not yet been any formal releases of sandboxfs, there
+currently are no prebuilt binaries available.  To use sandboxfs, you will
+have to build and install it from a fresh checkout of the GitHub tree.
+
+## From sources with Bazel
+
+The preferred mechanism to build sandboxfs is to use the
+[Bazel](http://bazel.build) build system.  The reason is that this provides
+a reproducible build environment with pinned versions of all required
+dependencies and also integrates well with various support tools used
+during installation and during development.
+
+To get started:
+
+1.  [Download and install Bazel](https://bazel.build/).
+
+1.  [Download and install Go](https://golang.org/).
+
+    *   If you don't want to do this by hand and would rather have Bazel
+        manage the installation of Go for you, edit the `WORKSPACE` file
+        and remove the `go_version = "host"` parameter from the
+        `go_register_toolchains` statement.
+
+1.  Run `bazel run //admin/install` to download all required dependencies,
```

Comment:
The install failed for me (on a Mac). Do we expect users to specify a custom prefix, or to have to use sudo, or ?

```
$ bazel run //admin/install
INFO: Found 1 target...
INFO: From SkylarkAction external/go_stdlib_darwin_amd64_cgo/bin/go [for host]:
# runtime/cgo
clang: warning: argument unused during compilation: '-pthread' [-Wunused-command-line-argument]
clang: warning: argument unused during compilation: '-gno-record-gcc-switches' [-Wunused-command-line-argument]
...
INFO: Elapsed time: 43.372s, Critical Path: 25.90s

INFO: Running command line: bazel-bin/admin/install/install
2017/11/20 13:08:24 cmd/sandboxfs/sandboxfs -> /usr/local/bin/sandboxfs (mode -rwxr-xr-x)
ERROR: failed to copy cmd/sandboxfs/sandboxfs to /usr/local/bin/sandboxfs: cannot open /usr/local/bin/sandboxfs for write: open /usr/local/bin/sandboxfs: permission denied
ERROR: Non-zero return code '1' from command: Process exited with status 1.
````

Reply:
Oh right... yes, you need root to copy onto `/usr/local` obviously, but `sudo bazel` is a pretty bad idea. I've updated the instructions accordingly.

Thanks for trying this instead of just reviewing the code! :-)

Reply:
Thanks!

I just tried following the new INSTALL.md, here's the new error I got:

```
$ bazel build //admin/install
INFO: Found 1 target...
Target //admin/install:install up-to-date:
  bazel-bin/admin/install/install
INFO: Elapsed time: 12.385s, Critical Path: 2.08s

$ ./bazel-bin/admin/install/install
2017/11/22 14:34:18 cmd/sandboxfs/sandboxfs -> /usr/local/bin/sandboxfs (mode -rwxr-xr-x)
ERROR: failed to copy cmd/sandboxfs/sandboxfs to /usr/local/bin/sandboxfs: cannot open cmd/sandboxfs/sandboxfs for read: open cmd/sandboxfs/sandboxfs: no such file or directory

$ sudo ./bazel-bin/admin/install/install
Password:
2017/11/22 14:34:24 cmd/sandboxfs/sandboxfs -> /usr/local/bin/sandboxfs (mode -rwxr-xr-x)
ERROR: failed to copy cmd/sandboxfs/sandboxfs to /usr/local/bin/sandboxfs: cannot open cmd/sandboxfs/sandboxfs for read: open cmd/sandboxfs/sandboxfs: no such file or directory
```

Reply:
OK... let's try again.

I've pushed a fix -- as a new commit to avoid the problem you observed before in GitHub and not being able to see what changed. However, this new commit will need fixing up into the right commit pre-merge.

Reply:
Okay, that worked!

```
$ sudo ./bazel-bin/admin/install/install
Password:
2017/11/25 19:33:24 bazel-bin/admin/install/install.runfiles/sandboxfs/cmd/sandboxfs/sandboxfs -> /usr/local/bin/sandboxfs (mode -rwxr-xr-x)
2017/11/25 19:33:24 bazel-bin/admin/install/install.runfiles/sandboxfs/AUTHORS -> /usr/local/share/doc/sandboxfs/AUTHORS (mode -rw-r--r--)
2017/11/25 19:33:24 bazel-bin/admin/install/install.runfiles/sandboxfs/CONTRIBUTING.md -> /usr/local/share/doc/sandboxfs/CONTRIBUTING.md (mode -rw-r--r--)
2017/11/25 19:33:24 bazel-bin/admin/install/install.runfiles/sandboxfs/CONTRIBUTORS -> /usr/local/share/doc/sandboxfs/CONTRIBUTORS (mode -rw-r--r--)
2017/11/25 19:33:24 bazel-bin/admin/install/install.runfiles/sandboxfs/INSTALL.md -> /usr/local/share/doc/sandboxfs/INSTALL.md (mode -rw-r--r--)
2017/11/25 19:33:24 bazel-bin/admin/install/install.runfiles/sandboxfs/LICENSE -> /usr/local/share/doc/sandboxfs/LICENSE (mode -rw-r--r--)
2017/11/25 19:33:24 bazel-bin/admin/install/install.runfiles/sandboxfs/NEWS.md -> /usr/local/share/doc/sandboxfs/NEWS.md (mode -rw-r--r--)
2017/11/25 19:33:24 bazel-bin/admin/install/install.runfiles/sandboxfs/README.md -> /usr/local/share/doc/sandboxfs/README.md (mode -rw-r--r--)
2017/11/25 19:33:24 bazel-bin/admin/install/install.runfiles/sandboxfs/cmd/sandboxfs/sandboxfs.1 -> /usr/local/share/man/man1/sandboxfs.1 (mode -rw-r--r--)
```

Reply:
Works in Linux too (that was OS X above)